benchmarking/example_classification.py

```diff
@@ -112,18 +115,30 @@
         execution_time = end - start
         print(f" - Time of fingerprints computing : {round(execution_time,2)}s")
         records[-1]["execution_time"] = execution_time
-        for classifier, clf_name, clf_kwargs in zip(
-            classifiers, clf_names, classifier_kwargs
-        ):
+
+        imputer = SimpleImputer(strategy="constant", fill_value=0)
+        X_fp_train = imputer.fit_transform(X_fp_train)
+        X_fp_valid = imputer.transform(X_fp_valid)
+        X_fp_test = imputer.transform(X_fp_test)
```

Comment:
Technically, this will not work properly for descriptors, just for fingerprints, which should never have missing values anyway. Maybe replace with median imputation?

Reply:
Sure